pkg/configuration/reader/model.go

```diff
@@ -34,8 +34,9 @@ type StreamList struct {
 }
 
 type Stream struct {
-	Name    string `yaml:"name"`
-	Conduit string `yaml:"conduit"`
+	Name            string `yaml:"name"`
+	Conduit         string `yaml:"conduit"`
+	IdentifierRange *Range `yaml:"identrifier-range"`
```

Comment:
Typo; identrifier-range

Reply:
fixed in "[Identifer range on LB](https://github.com/Nordix/Meridio/pull/175/commits/18700b454dbec02c9176a81bbb3c5637c9660caf)"